.evergreen/config.yml

```diff
@@ -1013,6 +1013,21 @@ functions:
           cd ${DRIVERS_TOOLS}/.evergreen/csfle
           ./kmstlsvenv/bin/python3 -u kms_http_server.py -v --ca_file ../x509gen/ca.pem --cert_file ../x509gen/${CERT_FILE} --port ${PORT} --require_client_cert
 
+  start-kms-mock-azure-imds-server:
+    - command: shell.exec
+      params:
+        shell: "bash"
+        script: |
+          ${PREPARE_SHELL}
+          cd ${DRIVERS_TOOLS}/.evergreen/csfle
+          . ./activate-kmstlsvenv.sh
+    - command: shell.exec
+      params:
+        background: true
+        script: |
+          cd ${DRIVERS_TOOLS}/.evergreen/csfle
+          ./kmstlsvenv/bin/python3 bottle.py fake_azure:imds
```

Comment:
Suggest adding `await_server "Azure" 8080` to `run-tests.sh` here: https://github.com/mongodb/mongo-go-driver/blob/eb83f20dc02a797b03aa1b4b988f98dd4fb0bd9d/.evergreen/run-tests.sh#L85

Using `background: true` will run this shell script in the background. Using `await_server` may prevent a race between the mock server starting and the test running.